aspnetcore/security/authentication/identity-configuration.md

```diff
@@ -12,6 +12,8 @@ uid: security/authentication/identity-configuration
 
 :::moniker range=">= aspnetcore-6.0"
 
+By [Damien Bowden](https://twitter.com/damien_bod)
+
```

Comment:
I did not write this, I should not get credit for this

Reply:
You reviewed it and I'm hoping you'll continue to review changes and consult. I can remove that line if you like until you've done more work on it.

Reply:
@Rick-Anderson will do, thanks for asking